recipes/qt/5.x.x/conanfile.py

```diff
@@ -661,6 +661,15 @@ def package(self):
         # symbols that are also in "Qt5Core.lib". It looks like there is no "Qt5Bootstrap.dll".
         for fl in glob.glob(os.path.join(self.package_folder, "lib", "*Qt5Bootstrap*")):
             os.remove(fl)
+        tools.save(os.path.join("lib", "cmake", "Qt5Core", "extras.cmake"),
+                    "set(Qt5Core_QMAKE_EXECUTABLE ${CMAKE_CURRENT_LIST_DIR}/../../../bin/qmake)\n"
+                    "set(Qt5Core_MOC_EXECUTABLE ${CMAKE_CURRENT_LIST_DIR}/../../../bin/moc)\n"
+                    "set(Qt5Core_RCC_EXECUTABLE ${CMAKE_CURRENT_LIST_DIR}/../../../bin/rcc)\n"
+                    "set(Qt5Core_UIC_EXECUTABLE ${CMAKE_CURRENT_LIST_DIR}/../../../bin/uic)")
```

Comment:
`qt5_wrap_ui` macro doesn't work, because it looks for `Qt5Widgets_UIC_EXECUTABLE`, not `Qt5Core_UIC_EXECUTABLE`. Could it be fixed in this pull request?